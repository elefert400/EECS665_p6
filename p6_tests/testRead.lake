int main(){
    int x;
    bool z;
    read x;
    read z;
    write x;
    write z;
}