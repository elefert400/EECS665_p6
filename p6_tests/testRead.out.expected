read from buffer: 3
read from buffer: 1
3
1
